packages/ui/v2/modules/booker/DatePicker.tsx

```diff
@@ -138,9 +138,10 @@ const DatePicker = ({
       onMonthChange(browsingDate.add(newMonth, "month"));
     }
   };
-
   const month = browsingDate
-    ? new Intl.DateTimeFormat(i18n.language, { month: "long" }).format(new Date(browsingDate.toISOString()))
+    ? new Intl.DateTimeFormat(i18n.language, { month: "long" }).format(
+        new Date(browsingDate.year(), browsingDate.month())
```

Comment:
so at browsing date that we pass is starting day of the month and if we change timezone it could happen a case that browsing date becomes 31 oct. 

but this so